ortho_seq_code/testdata/protein_test_data/CDR3H_padded_IC50.csv

```diff
@@ -0,0 +1,40 @@
+892.00

```

Comment:
good job on adding test data. is there any way you can add tests too?

Reply:
Yes! That's next on the list. :)